### PR TITLE
Site menu layout

### PR DESCRIPTION
**Pull request by raulraja**

### CODE REVIEW

**Comment by raulraja** [comment]
@AntonioMateoGomez there seem to be some issues with the menu opening and collapsing once there was more than one entry.

@rachelcarmena @franciscodr is the site automatically deploying when we merge on master?

File: docs/_data/sidebar.yml [hunk 1]
```diff
@@ -4,9 +4,85 @@ options:
...
+
+      - title: property
+        url: /docs/apidocs/compiler-plugin/arrow.meta.quotes/property.html
+
+      - title: propertyAccessor
+        url: /docs/apidocs/compiler-plugin/arrow.meta.quotes/property-accessor.html
+
+      - title: typeAlias
+        url: /docs/apidocs/compiler-plugin/arrow.meta.quotes/type-alias.html
+
```
Comment (ahinchman1) [inline docs/_data/sidebar.yml]: Nice, are these changes reflected in the kdocs? I was trying to figure out how to start adding pages, but I can just pull this down and work on a branch off yours

**Comment by franciscodr** [comment]
@raulraja Yes, there is a GitHub Actions workflow that generates and publishes the documentation after every pull request is merged into `master`. This is how we generated the site this morning.

**Comment by rachelcarmena** [comment]
@raulraja , about the issue with the menu, I don't know if it's the same as you are mentioning, but a problem occurs because of having duplicated entries. 

For instance, all these entries:

Screenshot from 2019-11-06 10-28-46

are related to:

Screenshot from 2019-11-06 10-28-55

So, the issue is solved when removing this part of the menu:

Screenshot from 2019-11-06 10-26-26

Shall we remove it?

File: docs/_data/sidebar.yml [hunk 2]
```diff
@@ -4,9 +4,85 @@ options:
...
+    nested_options:
+
+      - title: Config
+        url: /docs/apidocs/compiler-plugin/arrow.meta.dsl.config/-config-syntax/index.html
+
+      - title: Analysis
+        url: /docs/apidocs/compiler-plugin/arrow.meta.dsl.analysis/-analysis-syntax/index.html
+
+      - title: Resolution
+        url: /docs/apidocs/compiler-plugin/arrow.meta.dsl.resolve/-resolve-syntax/index.html
```
Comment (i-walker) [inline docs/_data/sidebar.yml]: @raulraja what options should we strive for in this part?

**Comment by rachelcarmena** [comment]
I removed my last comment because @AntonioMateoGomez is working on it right now :raised_hands:

**Comment by rachelcarmena** [comment]
Merging because the issue is solved with #82 :+1:

**Pull request merged**